usecase/cooking/cook.py

```diff
@@ -35,9 +35,15 @@ def set_services(self,
         self.calendar_service = calendar_service
         self.yelp_service = yelp_service
         self.spoonacle_service = spoonacle_service
-    
+
+    def set_default_services(self):
```

Comment:
To make the mock tests faster usecases shouldn't set the real services as default services in the `__init__`, but in an extra method `set_default_services` that will be called after construction, if wanted. 

Please consider this on your usecases @fionnfuchs @JannisTriesToCode .